### DISCUSSION TITLE
Remote crash logging

### DISCUSSION THREAD

**Discussion by GeckoEidechse** [R2Northstar/Northstar#688] [Research]
**The TL;DR is that by having a way to log crashes somewhere remotely, we will have an easier time to identify bugs and other issues as we'd no longer rely on players and server-hosters self-reporting them.**

Current discussion on Northstar Discord: https://discord.com/channels/920776187884732556/1275088689198010378/1275088689198010378

Previous discussions:
- https://discord.com/channels/920776187884732556/963562664410382346/963562781452419073 (requires contributor role to view due to the channel it was posted in)
- https://discord.com/channels/920776187884732556/1113976110850510979/1113976200486998066
